packages/tools/u-boot/package.mk

```diff
@@ -1,19 +1,20 @@
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
+#      This file is part of LibreELEC - http://www.libreelec.tv
```

Comment:
This file is part of LibreELEC - https://libreelec.tv

Reply:
Thanks. Fixed.